README.md

```diff
@@ -24,8 +24,8 @@ The Project building block facilitates the creation and engagement with micro-im
 
 # Supported Operating Systems
 
--   **Ubuntu (Recommended: Version 20 and above)** 
--   **Windows (Recommended: Version 11 and above)** 
+-   **Ubuntu (Recommended: Version 20 and above)**
+-   **Windows (Recommended: Version 11 and above)**
```

Comment:
@MallanagoudaB all the readme changes you made is breaking the format. not accepting this. While commiting read me changes avoid eslint

Reply:
i didn't made any changes for this even i tried with new repo getting the same issue will check that correctly